### PR TITLE
Release v4.0.0

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Membership Forms
- Fullprofile API Refactor
- T/S Date Picker Styling

# Review By (Date)
- EOD Thursday, 5/18 (I plan to merge this in Friday morning @ 9AM after getting confirmation that the API deployments are GTG. ETA for the API deployment is Thursday evening)

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to [`membership/join`](https://deploy-preview-668--stanford-alumni.netlify.app/membership/join)
2. Test the following form flows:
- Select `Myself` > From the payment options, select `One-time Payment` > Continue to form > Fill out form > Submit
- Select `Myself` > From the payment options, select `Installments` > Continue to form > Fill out form > Submit
- Select `Someone else` > Select a related contact > Continue to form > Fill out form > Submit
- Select `Someone else` > Select `Add new contact` > Continue to form > Fill out form > Submit
3. Navigate to [`membership/saacard`](https://deploy-preview-668--stanford-alumni.netlify.app/membership/saacard)
4. Login and verify that the correct card displays
5. Review code 👾 

# Associated Issues and/or People
- ADAPTSM-74

### CODE REVIEW

File: src/components/cards/membershipCard.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    fetchImages(false);
+    memberships.forEach((membership) => {
+      if (
+        membership.membershipStatus?.includes('Active') &&
+        membership.membershipGroup?.includes('SAA') &&
+        membership.membershipAffiliation?.includes('Alum')
+      ) {
+        fetchImages(true, SaaLogoWhite, SaaCardBg);
+        setBgColor(
+          'su-bg-gradient-to-b su-from-[#8E1515] su-to-digital-red-light'
```
Comment (yvonnetangsu) [inline src/components/cards/membershipCard.js:41]: `#8E1515` hmmm so strange, I wonder if they meant for this to be cardinal red which is `#8C1515` 🤔  Is this in the mock @rebeccahongsf ? Don't worry if it is I'm ok with updating after the release 🤣

**Comment by yvonnetangsu** [comment]
@rebeccahongsf Looks like a redirect is active currently if I click on membership/register. Is there another way to see that page? 😄

File: src/components/cards/membershipCard.js (hunk 1 above)
Comment (rebeccahongsf) [reply src/components/cards/membershipCard.js:41]: Ooo great question 🤔 I cannot seem to find that color in the [redesign of the membership cards](https://www.figma.com/file/c8Zv388RgBwDcSPtBTZhc1/Digital-Membership-Cards?type=design&node-id=1%3A61&t=r2xC9cuAeX4bORFn-1)

I'll be making some changes to `dev` so I can update it to Cardinal red 😎

Comment (yvonnetangsu) [reply src/components/cards/membershipCard.js:41]: Not a big priority @rebeccahongsf, but the [ ] bracket caught my eyes hahaha 🤣

Comment (yvonnetangsu) [reply src/components/cards/membershipCard.js:41]: And I do realize this is collab work with our vendors who aren't as familiar with our colors 👍🏼

**Comment by yvonnetangsu** [comment]
Thanks @rebeccahongsf!

**Comment by yvonnetangsu** [comment]
@rebeccahongsf You mentioned leaving comments here so I'm going to do that 😄 

1. I noticed that the "Selected" indicator at the top right corner for these two cards are positioned differently. Without looking at the code, I'm suspecting it's due to it aligning to the left side of the content below, and since the first card has narrower content, the "Selected" text is more inset compared to the other one?

Screenshot 2023-05-18 at 11 51 44 AM
Screenshot 2023-05-18 at 11 51 39 AM

2. It seems the select drop down in the GG form doesn't have a focus state (try tabbing to a select field)
3. If we could use the hand cursor to the checkbox/radio buttons that'd be awesome 😄

**Comment by rebeccahongsf** [comment]
> @rebeccahongsf You mentioned leaving comments here so I'm going to do that 😄
> 
> 1. I noticed that the "Selected" indicator at the top right corner for these two cards are positioned differently. Without looking at the code, I'm suspecting it's due to it aligning to the ~left~ (lol I meant right side) side of the content below, and since the first card has narrower content, the "Selected" text is more inset compared to the other one?
> 
> Screenshot 2023-05-18 at 11 51 44 AM Screenshot 2023-05-18 at 11 51 39 AM
> 
> 2. It seems the select drop down in the GG form doesn't have a focus state (try tabbing to a select field)
> 3. If we could use the hand cursor to the checkbox/radio buttons that'd be awesome 😄

These details definitely matter 😃 I got PR #671 together to fix those issues. Thank you @yvonnetangsu !

**Pull request closed without merging**